x/msgfee/init.go

```diff
@@ -0,0 +1,43 @@
+package msgfee
+
+import (
+	"fmt"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/errors"
+)
+
+// Initializer fulfils the Initializer interface to load data from the genesis
+// file
+type Initializer struct{}
+
+var _ weave.Initializer = (*Initializer)(nil)
+
+// FromGenesis will parse initial account info from genesis and save it to the
+// database
+func (*Initializer) FromGenesis(opts weave.Options, db weave.KVStore) error {
+	type msgfee struct {
+		MsgPath string    `json:"msg_path"`
+		Fee     coin.Coin `json:"fee"`
```

Comment:
pp: As mentioned earlier `fee` is a bit overloaded. Maybe this can become the product catalog and this is the price?

Reply:
Continued in https://github.com/iov-one/weave/pull/376#discussion_r262918072